homeassistant/components/lyric/config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Config flow for Honeywell Lyric."""
+import asyncio
+import logging
+
+import async_timeout
+from lyric import Lyric
+
+from homeassistant import config_entries
+from homeassistant.components.http import HomeAssistantView
+from homeassistant.config_entries import ConfigFlow
+from .const import (
+    AUTH_CALLBACK_NAME,
+    AUTH_CALLBACK_PATH,
+    DOMAIN,
+    CONF_CLIENT_ID,
+    CONF_CLIENT_SECRET,
+    CONF_LYRIC_CONFIG_FILE,
+    DATA_LYRIC_CONFIG,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class LyricFlowHandler(ConfigFlow):
```

Comment:
We can do this nowadays instead of using the decorator:

```suggestion
class LyricFlowHandler(ConfigFlow, domain=DOMAIN):
```